pkg/tinydtls/Makefile.include

```diff
@@ -0,0 +1 @@
+INCLUDES += -I$(RIOTBASE)/pkg/tinydtls -I$(RIOTBASE)/pkg/tinydtls/tinydtls
```

Comment:
`$(RIOTBASE)/pkg/tinydtls` does not contain any header files, so why add it to the include path?